access_win.go

```diff
@@ -7,8 +7,9 @@ import (
 	"strings"
 )
 
-func access(path string) error {
-	if strings.Contains(path, ".exe") {
+func isExecutableOnPlatform(path string) error {
+	if strings.HasSuffix(path, ".exe") || strings.HasSuffix(path, ".cmd") ||
```

Comment:
Win32 API would overkill. I added .cmd and .bat extensions. adb is a .exe but the sdk has a few .bat file so this will be future proof.